cheatsheets/Password_Storage_Cheat_Sheet.md

```diff
@@ -2,7 +2,7 @@
 
 ## Introduction
 
-As the majority of users will re-use passwords between different applications, it is important to store passwords in a way that prevents them from being obtained by an attacker, even if the application or database is compromised. As with most areas of cryptography, there are many different factors that need to be considered, but fortunately, the majority of modern languages and frameworks provide built-in functionality to help store passwords, which handles much of the complexity.
```

Comment:
I made this paragraph more succinct by removing things that seemed unnecessary.

- Regardless of how often passwords are reused, hashing is important
- Seemed like a lot of words to simply state that modern languages should have libraries to help you